modules/microshift-cni.adoc

```diff
@@ -0,0 +1,301 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-networking.adoc
+
+:_content-type: CONCEPT
+[id="microshift-cni_{context}"]
+= About the OVN-Kubernetes CNI network provider
+
+//Q: this module is not done; included for the sake of the build
+//[IMPORTANT]
+//====
+//The default CNI configuration is intended to match the developer environment described in MicroShift Development Environment on RHEL 8.
+//====
+//Q: What's the default production configuration?
+
+{product-title} uses OVN-Kubernetes, a virtualized network for pods and services. The OVN-Kubernetes Container Network Interface (CNI) plug-in is the network provider for the cluster. OVN-Kubernetes is based on Open Virtual Network (OVN) and provides an overlay-based networking implementation.
+
+A cluster that uses the OVN-Kubernetes network provider also runs Open vSwitch (OVS) on the node. OVN configures OVS on the node to implement the declared network configuration. OVS runs as a systemd service on the {product-title} node. The OVS RPM package is installed as a dependency to the `microshift-networking` RPM package. With {product-title}, each deployment requires configuration. OVS is started immediately when the `microshift-networking` RPM is installed.
+//Q: can customers copy network settings from a first node onto others, or do them all at once?
+//Q: is this conceptually correct = one microshift install can create several single-node deployments? or is it one-to-one and a new microshift instance has to be installed for every single node deployment?
+
+By default, three performance optimizations are applied to OVS services to minimize resource consumption:
+
+* CPU affinity to `ovs-vswitchd.service` and `ovsdb-server.service`
+* `no-mlockall` to `openvswitch.service`
+* Limit handler and `revalidator` threads to `ovs-vswitchd.service`
+
+== Network features
+A wide range of networking features are available with OVN-Kubernetes, such as:
+
+* Kubernetes network policy
+* Dynamic node IP
+* Cluster network on specified host interface
+* Secondary gateway interface
+* Dual stack
+* Service idling (disabled)
+* Egress firewall/QOS/IP (disabled)
```

Comment:
```suggestion
//* Service idling (disabled)
//* Egress firewall/QOS/IP (disabled)
```

Those are features of OVN which are disabled and the end user cannot enable them. So I would leave them out of the docs, or at least commented so the customers can't see this.